.gitignore

```
@@ -1,2 +1,3 @@
 /dist
 /node_modules
+implementations/boon/target/
```

```suggestion
/implementations/boon/target/
```

Just to be explicit?